packages/astro/src/events/index.ts

```diff
@@ -1 +1,16 @@
+import { AstroTelemetry } from '@astrojs/telemetry';
+import { ASTRO_VERSION } from '../core/util.js';
+
+function getViteVersion() {
+	try {
+		const { version } = require('vite/package.json');
```

Comment:
`require` isn't defined! Astro uses `"type": "module"`.

Reply:
it appears that TS handles this for you: https://unpkg.com/browse/astro@1.0.0-beta.58/dist/events/session.js

Fixed it anyway

Reply:
Thanks I hate it.